apps/client/src/app/userId/[id]/page.tsx

```diff
@@ -0,0 +1,17 @@
+import QRCode from "react-qr-code";
+
+export default function Index({ params }: {
+    params: {id: string}
+}) {
+    return <div className='m-10'>
+        <h1 className='text-center mb-8' >My QR-code </h1>
+        <div>
+            <QRCode
+                size={256}
+                style={{ height: "auto", maxWidth: "100%", width: "100%" }}
```

Comment:
Check if it's possible to replace those styles to tailwind css syntax. And if it's possible, please change it, so we will have tailwind styles everywhere

Reply:
It is possible. I changed.

Reply:
done